### ISSUE TITLE
Version 0.14.6 keeps crashing even before launching

### ISSUE THREAD

**Issue by ArzuHrpt** [tachiyomiorg/tachiyomi#9439]
### Steps to reproduce

1. Tap on icon
2. App crashes only after icon flashes on screen

### Expected behavior

App should launch properly when I try to open it

### Actual behavior

App keeps crashing regardless of how many times I try 

### Tachiyomi version

0.14.6-5720

### Android version

Android 10

### Device

Samsung Galaxy Note 9

### Other details

My software is up to date, along with Google Chrome and Webview. I made sure to check it several times. I restarted my cellphone and i cleared cache but no matter what I tried it didn't solve the problem. It's been like this for several days now and only thing i didn't do is deleting the data of the app or completely uninstalling and reinstalling it

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have gone through the [FAQ](https://tachiyomi.org/help/faq/) and [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by arkon** [comment]
Manually download and install the update from https://tachiyomi.org/download/

**Comment by ArzuHrpt** [comment]
@arkon i already did it but i have a separate and new app on my device now. All my data is still sitting on old one. Is there no way to save it?

**Comment by arkon** [comment]
You need to download the preview build if you want to upgrade the preview app. Both are linked on that page.

**Comment by ArzuHrpt** [comment]
@arkon Mine was a stable version though, should I still go for  preview one?

**Comment by arkon** [comment]
5720 is a preview build, not a stable build. If you want to switch, upgrade first, then create a backup and restore it in stable.
